akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1245,7 +1245,7 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
 
   /**
    * Terminate processing (and cancel the upstream publisher) after predicate
-   * returns false for the first time, including the first failed element iff inclusive is true
+   * returns false for the first time, including the first failed element if inclusive is true
```

Comment:
This was not a typo but a 'math-ism' (https://en.wikipedia.org/wiki/If_and_only_if). I like user simpler language though. Perhaps really spell out: "Terminate processing (and cancel the upstream publisher) after predicate returns false for the first time. When 'inclusive' is true, also include the element for which the predicate returned 'false'."

Reply:
That was completely lost on me! It certainly looks like a typo to us mere mortals.

Reply:
Jup! Could you either revert back to iff or, even better, to a more clear (but verbose) description like I proposed?